start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/sdk/alfred/compose/AlfredSdkComposeProjectGenerationConfiguration.java

```diff
@@ -0,0 +1,92 @@
+package eu.xenit.alfred.initializr.start.sdk.alfred.compose;
+
+import eu.xenit.alfred.initializr.start.build.BuildCustomizer;
+import eu.xenit.alfred.initializr.start.build.root.gradle.RootGradleBuild;
+import eu.xenit.alfred.initializr.generator.docker.compose.DockerComposeCustomizer;
+import eu.xenit.alfred.initializr.generator.docker.compose.DockerComposeLocationStrategy;
+import eu.xenit.alfred.initializr.start.sdk.alfred.compose.config.ComposeUpGradleTaskConfiguration;
+import eu.xenit.alfred.initializr.start.sdk.alfred.compose.config.ComposeUpGradleTaskConfigurationCustomizer;
+import eu.xenit.alfred.initializr.start.sdk.alfred.compose.config.DockerComposeGradlePluginConfiguration;
+import eu.xenit.alfred.initializr.start.sdk.alfred.compose.config.DockerComposeGradlePluginConfigurationCustomizer;
+import io.spring.initializr.generator.buildsystem.gradle.GradleBuildSystem;
+import io.spring.initializr.generator.condition.ConditionalOnBuildSystem;
+import io.spring.initializr.generator.project.ProjectGenerationConfiguration;
+import io.spring.initializr.generator.project.ProjectDescription;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.boot.util.LambdaSafe;
+import org.springframework.context.annotation.Bean;
+
+@ProjectGenerationConfiguration
+@ConditionalOnBuildSystem(GradleBuildSystem.ID)
+public class AlfredSdkComposeProjectGenerationConfiguration {
+
+    @Bean
+    public DockerComposeCustomizer basicAlfrescoComposeCustomizer(ProjectDescription projectDescription) {
+        return new AlfrescoBaseLayerComposeCustomizer(projectDescription);
+    }
+
+    @Bean
+    public DockerComposeLocationStrategy dockerComposeLocationStrategy() {
+        return new DockerComposeLocationStrategy(Paths.get("docker-compose"));
```

Comment:
Can we make this configurable?

Reply:
This *IS* the configuration ? :-)

Alfresco Maven SDK can override this by providing it's own `DockerComposeLocationStrategy` ?